app/lib/publisher/backend.dart

```diff
@@ -40,17 +41,6 @@ class PublisherBackend {
     return (await _db.lookup<Publisher>([pKey])).single;
   }
 
-  /// Checks whether the current authenticated user has admin role of the
-  /// publisher, and executes [fn] if it does.
-  /// Otherwise, it throws [AuthorizationException].
-  /// TODO: remove this and migrate to use withPublisherAdmin
-  Future<R> _withPublisherAdmin<R>(
```

Comment:
Hmm, can we have `withPublisherAdmin` call `withAuthenticatedUser`? Or does it need to be separate, it seems like two wrappers always used together...

We could also change these to: `final u = await ensureAuthenticatedUser();` and `await ensurePublisherAdmin(u);` just to reduce indentation... I guess the with-pattern is a bit safer though...

Reply:
Removed this from the PR, will do the other refactor separately.